### PR TITLE
removed !

### PR DESCRIPTION
**Pull request by Newmang314**
In this, I removed the ! because it was unneccessary, and because of improvisation because of reviews.

### CODE REVIEW

**Comment by Newmang314** [comment]
Wait, I forgot to add the al error, my bad.

File: index.html
Comment (jayminard) [inline index.html:1]: I like the changes. Lets fix the spelling in the H2 tag and we should be good.

**Review by jayminard — CHANGES_REQUESTED** [review]
Please change Al to All in H2 tag

File: index.html
Comment (Newmang314) [reply index.html:1]: Okay.

**Pull request merged**